ngx-tools/keycodes/README.md

```diff
@@ -1,51 +1,57 @@
 <h1>Keycodes</h1>
 
-**Import from:** `@terminus/ngx-tools/keycodes`
-
-```typescript
-// Example usage:
-import { KEYS } from '@terminus/ngx-tools/keycodes';
-import { dispatchKeyboardEvent } from '@terminus/ngx-tools/testing';
+A collection of key codes to avoid duplicate strings throughout your codebase.
 
-dispatchKeyboardEvent(myElementRef, KEYS.ESCAPE);
-
-if (event.code === KEYS.ESCAPE.code) {
-  console.log('Got the correct key code');
-}
-```
+**Import from:** `@terminus/ngx-tools/keycodes`
 
 
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 **Table of Contents**
 
+- [Keys](#keys)
 - [Available Key Codes](#available-key-codes)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-## Available Key Codes
+## Keys
 
 [[source]](src/keycodes.const.ts)
 
-| Name          |
-|---------------|
-| `UP_ARROW`    |
-| `DOWN_ARROW`  |
-| `RIGHT_ARROW` |
-| `LEFT_ARROW`  |
-| `PAGE_UP`     |
-| `PAGE_DOWN`   |
-| `HOME`        |
-| `END`         |
-| `ENTER`       |
-| `SPACE`       |
-| `TAB`         |
-| `BACKSPACE`   |
-| `DELETE`      |
-| `A`           |
-| `Z`           |
-| `ZERO`        |
-| `NINE`        |
-| `COMMA`       |
-| `ESCAPE`      |
+The `KEYS` collection contains the `code` and `keyCode` for all supported keys.
+
+```typescript
+import { KEYS } from '@terminus/ngx-tools/keycodes';
+import { dispatchKeyboardEvent } from '@terminus/ngx-tools/testing';
+
+KEYS.ENTER.code    // 'Enter'
+KEYS.ENTER.keyCode // 13
+
+// Example usage:
+dispatchKeyboardEvent(myElementRef, KEYS.ENTER.keycode);
+```
+
+
+## Available Key Codes
+
+[[source]](src/keycodes.const.ts)
 
+| Name                  |
+|-----------------------|
+| `BACKSPACE`           |
+| `TAB`                 |
+| `ENTER`               |
+| `ESCAPE`              |
+| `SPACE`               |
+| `PAGE_UP`             |
+| `PAGE_DOWN`           |
+| `END`                 |
+| `HOME`                |
+| `LEFT_ARROW`          |
+| `UP_ARROW`            |
+| `RIGHT_ARROW`         |
+| `DOWN_ARROW`          |
+| `DELETE`              |
+| `ZERO` through `NINE` |
+| `A` through `Z`       |
```

Comment:
NOTE: I was trying to avoid being overly verbose and used the `X through Y` approach.. is it still clear that each item is still available?